roles/rsyslog/tasks/main.yml

```diff
@@ -1,3 +1,7 @@
+- name: Include top vars
+  include_vars:
+    file: "{{ role_path | dirname | dirname }}/vars/main.yml"
```

Comment:
What's the point of this change? `vars/main.yml` gets included automatically in any role.

Reply:
Without this include_vars, it fails as follows. And default_logging_provide is needed as described in https://github.com/linux-system-roles/logging/pull/110#discussion_r419597869.
```
task path: /home/nhosoi/linux-system-roles/logging/roles/rsyslog/handlers/main.yml:2
fatal: [/home/nhosoi/Downloads/CentOS-Stream-GenericCloud-8-20200113.0.x86_64.qcow2]: FAILED! => {
    "msg": "The conditional check 'logging_provider | d(default_logging_provider) == 'rsyslog'' failed. The error was: error while evaluating conditional (logging_provider | d(default_logging_provider) == 'rsyslog'): 'default_logging_provider' is undefined\n\nThe error appears to be in '/home/nhosoi/linux-system-roles/logging/roles/rsyslog/handlers/main.yml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: restart rsyslogd\n  ^ here\n"
}

```